hello-spring/src/main/resources/api.yaml

```diff
@@ -0,0 +1,52 @@
+openapi: 3.0.3
+info:
+  title: Hello Spring
+  description: A minimal "Hello, world" FOLIO module written in Java, inspired by the `hello-vertx` example.
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
```

Comment:
If going to also declare license here, then should be the same as in the POM (i.e. spdx.org).